providers/mongodb.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-shadow */
+let db;
+const config = {
+  moduleName: "mongo",
+  enabled: true,
+  dbName: "Komada",
+  dbURL: "mongodb://localhost:27017/",
+};
+
+function connect() { db.connect(`${config.dbURL}${config.dbName}`); }
+
+// Collection Methods. Collections are implicitly created with document methods regardess.
+
+/**
+ * Create a collection within a DB. Options may be specfied, refer to MongoDB docs.
+ *
+ * @param {string} name Name of the Collection to crate
+ * @param {Object} [options] Object containing various options for the created Collection
+ * @returns {Promise} Returns a promise containing the created Collection.
```

Comment:
This returns a `Promise`, yeah, but what does that Promise return after being resolved? If it returns nothing, `Promise<void>`.